ui/StatusQ/sandbox/sandboxapp.cpp

```diff
@@ -47,15 +47,15 @@ void SandboxApp::restartEngine()
         qDebug() << "Restarting QML engine";
 
     if (m_engine)
-        delete m_engine;
+        m_engine.release();
 
-    m_engine = new QQmlApplicationEngine();
+    m_engine = std::make_unique<QQmlApplicationEngine>();
```

Comment:
You could just use `m_engine.reset(new QQmlApplicationEngine)` w/o the `release()` call above

Reply:
indeed. That's what blind replacing does with us 😄

Reply:
And btw, `release()` does _not_ delete the raw pointer

Reply:
https://en.cppreference.com/w/cpp/memory/unique_ptr/release